Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,39 @@ label monika_language_nuances:
     m 1tsu "So good day to you, [player].{w=0.3} {nw}"
     extend 1hub "Ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='monika_fear',
+            prompt="Fear",
+            category=['monika'],
+            conditional=("renpy.seen_label('monika_soft_rains')"),
+            action=EV_ACT_RANDOM
+        )
+    )
+
+label monika_worst_fear:
+    m 3eud "Say, [player]..."
+    m 1eud "This is kind of a weird question, but... {w=0.5} Is there something you're afraid of?"
```

Comment:
```suggestion
    m 1eud "This is kind of a weird question, but...{w=0.5}is there something you're afraid of?"
```
Since this should all be one sentence, no space